utils/translationKeys.txt

```diff
@@ -0,0 +1,106 @@
+Bar Width                                                                                                    // /DefaultEditor.js:139
+Bars                                                                                                         // /components/containers/TraceMarkerSection.js:20
+Blank                                                                                                        // /DefaultEditor.js:166
+Border Color                                                                                                 // /DefaultEditor.js:134
+Border Width                                                                                                 // /DefaultEditor.js:133
+Bottom                                                                                                       // /DefaultEditor.js:213
+Box Padding                                                                                                  // /DefaultEditor.js:151
+Box Width                                                                                                    // /DefaultEditor.js:143
+Canvas                                                                                                       // /DefaultEditor.js:174
+Center                                                                                                       // /DefaultEditor.js:265
+Color                                                                                                        // /DefaultEditor.js:118
+Common Case: An 'All' tab might display this message because the X and Y tabs contain different settings.    // /lib/constants.js:24
+Connect                                                                                                      // /DefaultEditor.js:165
+Connect Gaps                                                                                                 // /DefaultEditor.js:162
+Continue                                                                                                     // /components/widgets/text_editors/MultiFormatTextEditor.js:197
+Continuing will convert your LaTeX expression into raw text.                                                 // /components/widgets/text_editors/MultiFormatTextEditor.js:121
+Continuing will convert your note to LaTeX-style text.                                                       // /components/widgets/text_editors/MultiFormatTextEditor.js:116
+Continuing will remove your expression.                                                                      // /components/widgets/text_editors/MultiFormatTextEditor.js:126
+Custom                                                                                                       // /DefaultEditor.js:179
+Custom colors                                                                                                // /components/widgets/ColorPicker.js:52
```

Comment:
Looks like we've generally capitalized all words of titles, so `Custom Colors`? `Default Colors`? Less sure about `Enter Link URL`...